app/Jobs/ValidateCSV.php

```diff
@@ -136,6 +136,9 @@ private function checkFieldErrors($mismatch): ?string
                 'prohibited_if:wikidata_value,',
                 'max:' . $rules['meta_wikidata_value']['max_length'],
                 'regex:' . $rules['meta_wikidata_value']['format']
+            ],
+            'type' => [
+                'string'
```

Comment:
I did add the `in` rule before and it was failing the import, but it must have been for another reason because I've added it again and it works!! :tada: Thank for your observation !!